QSB/Finder.cs

```diff
@@ -0,0 +1,163 @@
+﻿using Newtonsoft.Json;
+using QSB.Animation;
+using QSB.Events;
+using QSB.Utility;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+namespace QSB
+{
+    public static class Finder
```

Comment:
From the name I'd assume this class would be just like Outer Wilds'  `Locator`, but it doesn't seem to be that at all. Everything here is related to players or player states, so you should probably name the class accordingly.

And I'm sure Alek would hate this class, because you're using a static class that has inner state (`playerList`). But I'll leave it to him to decide on what's best here.